packages/react-storage/src/components/StorageBrowser/components/Table.tsx

```diff
@@ -46,31 +53,33 @@ export const Table = ({ headers, rows }: TableProps): React.JSX.Element => {
           ) : null}
         </TableHeadElement>
         <TableBodyElement className={`${STORAGE_BROWSER_BLOCK}__table-body`}>
-          {rows?.map(({ key, content }) => (
-            <TableRowElement
-              key={key}
-              className={`${STORAGE_BROWSER_BLOCK}__table-row`}
-            >
-              {content.map(({ key, content, type }) => {
-                return type === 'header' ? (
-                  <TableHeaderElement
-                    key={key}
-                    className={`${STORAGE_BROWSER_BLOCK}__table-header`}
-                    role="rowheader"
-                  >
-                    {content}
-                  </TableHeaderElement>
-                ) : (
-                  <TableDataCellElement
-                    key={key}
-                    className={`${STORAGE_BROWSER_BLOCK}__table-data-cell`}
-                  >
-                    {content}
-                  </TableDataCellElement>
-                );
-              })}
-            </TableRowElement>
-          ))}
```

Comment:
This portion is not changed, just moved into conditional rendering based on loading status